test/e2e/parameters/parameters.test.js

```diff
@@ -119,5 +119,5 @@ test('should parse parameters from jsdoc-example', async () => {
     },
   };
   const result = await processSwagger(options);
-  expect(result).toEqual(expected);
+  expect(result.swaggerObject).toEqual(expected);
```

Comment:
I would also add the methods in this test, to validate that we are returning methods 😄